MCprep_addon/spawner/mcmodel.py

```diff
@@ -350,11 +360,22 @@ def add_model(
 				# Cake and cake slices don't store the UV keys
 				# in the JSON model, which causes issues. This
 				# workaround this fixes those texture issues
+				uv_from = env.json_data.get("block_model_uv")
+				stored_uv_blocks = ("hopper", "cauldron", "scaffolding", "composter")
```

Comment:
I assume this is to fix issues such as those mentioned in https://github.com/Moo-Ack-Productions/MCprep/issues/649#issuecomment-2642253946?